src/ros_filter.cpp

```diff
@@ -815,7 +815,7 @@ void RosFilter<T>::loadParams()
   // Try to resolve tf_prefix
   std::string tf_prefix = "";
   std::string tf_prefix_path = "";
-  this->declare_parameter("tf_prefix");
+  this->declare_parameter<std::string>("tf_prefix");
```

Comment:
I misunderstood how the new static type enforcement works. This code will throw an exception at this line if "tf_prefix" is not provided, but we want it to me optional.

Reply:
I've addressed this by handling NoParameterOverrideProvided exceptions when no parameters are provided: 3326e80